NAMESPACE

```diff
@@ -222,13 +222,16 @@ importFrom(decontam,isNotContaminant)
 importFrom(dplyr,"%>%")
 importFrom(dplyr,arrange)
 importFrom(dplyr,desc)
+importFrom(dplyr,filter)
 importFrom(dplyr,group_by)
 importFrom(dplyr,left_join)
 importFrom(dplyr,mutate)
 importFrom(dplyr,n)
 importFrom(dplyr,rename)
 importFrom(dplyr,select)
 importFrom(dplyr,tally)
+importFrom(reshape2,dcast)
+importFrom(reshape2,melt)
```

Comment:
If the motivation for a new dependency is speed, could we then consider `data.table::dcast` and `data.table::melt`?

They are even [faster, and also optimized for very large data sets](https://cran.r-project.org/web/packages/data.table/vignettes/datatable-reshape.html).